modules/tinymce/src/themes/silver/main/ts/backstage/Backstage.ts

```diff
@@ -23,9 +23,6 @@ import { HeaderBackstage, UiFactoryBackstageForHeader } from './HeaderBackstage'
 import { init as initStyleFormatBackstage } from './StyleFormatsBackstage';
 import { UiFactoryBackstageForUrlInput, UrlInputBackstage } from './UrlInputBackstage';
 
-// INVESTIGATE: Make this a body component API ?
```

Comment:
After looking over this it should have been `BodyComponent` so have made that change.